CHANGES/9517.misc

```diff
@@ -0,0 +1 @@
+Added ``signing_service`` for (apt-)repository in Complex-MigrationPlan.
```

Comment:
I would consider this a feature, rather than just a misc.
User defined migration plan is being extended with this change.

Somehting like `Added an ability to use a manually pre-created Pulp 3 SigningService during pulp_deb repository migration. A migration plan now accepts ``signing_service`` key for each repository.`

What do you think?

Reply:
Sounds good, done :smiley: